dev-requirements.txt

```
@@ -1,6 +1,6 @@
 git+https://github.com/dbt-labs/dbt-adapters.git@main
 git+https://github.com/dbt-labs/dbt-adapters.git@main#subdirectory=dbt-tests-adapter
-git+https://github.com/dbt-labs/dbt-common.git@main
+git+https://github.com/dbt-labs/dbt-common.git@dbeatty/9840-quiet-show
```

Make sure to restore the original branch prior to final merge:
```suggestion
git+https://github.com/dbt-labs/dbt-common.git@main
```